dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -49,6 +52,7 @@
 
     private static final String CASE_FILE = "case-file";
     private static final String DYNAMIC_MODEL_FILE = "dynamic-model-file";
```

Comment:
Rename this constant DYNAMIC_MODELS_FILE

Reply:
Done